pytorch/training/docker/2.2/py3/cu121/Dockerfile.gpu

```diff
@@ -374,10 +374,6 @@ ARG GDRCOPY_VERSION
 # SageMaker Profiler Binary
 ARG SMP_URL=https://smppy.s3.amazonaws.com/pytorch/cu121/smprof-0.3.334-cp310-cp310-linux_x86_64.whl
 
-# Selectively enable PyTorch Verbose Logging
```

Comment:
This is removed due to this [thread](https://amzn-aws.slack.com/archives/C06EZT2RB0B/p1707771786863799)